src/controller/model/endpoint.ts

```diff
@@ -285,8 +285,9 @@ class Endpoint extends Entity {
 
         // send without queueing if sendPolicy is 'immediate' or if the device has no timeout set
         if (request.sendPolicy === 'immediate' || !device.pendingRequestTimeout) {
+            logger.debug(logPrefix + `send ${frame.command.name} request immediately (sendPolicy=${options.sendPolicy})`, NS);
+
             if (device.pendingRequestTimeout > 0) {
```

Comment:
Empty `if` statement.

Reply:
Whoops, I shouldn't have committed this.